frontend/server/src/DAO/QualityNominations.php

```diff
@@ -123,6 +123,70 @@ public static function reviewerHasQualityTagNominatedProblem(
         ) > 0;
     }
 
+    /**
+    * Returns the contents and id of a nomination for a given problem and user.
+    * @return array{contents: array{quality_seal: bool, tag: string}, qualitynomination_id: int}
```

Comment:
Igual, acá faltaría indicar que esto puede regresar `null` de acuerdo al comentario anterior
```suggestion
    * @return array{contents: array{quality_seal: bool, tag: string}, qualitynomination_id: int}|null
```